scripts/publish.sh

```diff
@@ -8,7 +8,7 @@ check() {
         printf "Export %s finished properly\n." "$@"
     else
         printf "Export %s failed. Exited with %s\n" "$@" "$?"
-        printf "------------------\n\n"
+        printf "%s------------------\n\n"
```

Comment:
Shouldn't this be `printf '%s\n\n' "------------------"`?

Reply:
@rebeccajfriedman True, fixed
> Shouldn't this be printf '%s\n\n' "------------------"?